pkgs/development/ada-modules/gnatcoll/bindings.nix

```diff
@@ -42,6 +43,24 @@ stdenv.mkDerivation rec {
     sha256 = "00aakpmr67r72l1h3jpkaw83p1a2mjjvfk635yy5c1nss3ji1qjm";
   };
 
+  patches = [
+    (fetchpatch {
```

Comment:
`fetchpatch2` is the new and improved version (`fetchpatch` is there for compatibility).